dashboard/client/src/api/functionsApi.js

```diff
@@ -52,7 +51,8 @@ class FunctionsApi {
       if (this.queryPrettyUrl) {
         endpoint = this.prettyDomain
           .replace('user', user)
-          .replace('function', shortName);
+        endpoint = replaceFunctionEnding(endpoint, shortName)
```

Comment:
`replaceFunctionEnding` <- this may be what the code does, but isn't it more like `formatFunctionURL` or `getFunctionURL`?